node/commons/compatibility-layer.ts

```diff
@@ -157,7 +157,7 @@ const fillSearchItemWithSimulation = (searchItem: SearchItem, orderFormItems: Or
         return
       }
 
-      seller.commertialOffer.AvailableQuantity = orderFormItem?.availability === 'withoutPriceFulfillment' ? 0 : seller.commertialOffer.AvailableQuantity
+      seller.commertialOffer.AvailableQuantity = orderFormItem?.availability === 'available' ? 10000 : seller.commertialOffer.AvailableQuantity
```

Comment:
When the availability is `withoutPriceFulfillment ` we still need to return `0` here.

It would be like this:
case `withoutPriceFulfillment`, returns `0`
case `available` returns `1000`
default `AvailableQuantity`

Btw, why is `10000` hardcoded? Shouldn't we use `AvailableQuantity`?

Reply:
Actually, we should actually use `available ? availableQuantity : 0`, I'll change it